src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Color {
+    ED,
```

Comment:
There seems to be a typo in the color name 'ED'. It might be intended to be 'RED'. Please verify and correct it if necessary.